hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -116,6 +116,15 @@ if {$env(DUT_TOP) == "pinmux_tb"} {
   clock -rate {tl_i, cio_d_i, cio_dp_i, cio_dn_i, cio_sense_i} clk_i
   reset -expr {!rst_ni !rst_aon_ni}
 
+} elseif {$env(DUT_TOP) == "clkmgr"} {
+  clock clk_main_i
+  clock clk_i -both_edges
+  clock clk_io_i -factor 1
+  clock clk_usb_i -factor 1
+  clock clk_aon_i -factor 2
+  clock -rate -default clk_i
```

Comment:
The clock ratios are not really reflecting the top_earlgrey one because formal does not really designed to verify different clock domains. If I use the original ratios, the formal tool returns an error saying the clock is too complex and could not handle them.
So I mainly just identify different clocks and resets for the design.

Reply:
I think this is contingent on the correctness of the CDC flops, which has been beat up to a pulp in many other ways, so this would be proof enough for me.